src/main/kotlin/org/rust/lang/core/types/Substitution.kt

```diff
@@ -22,7 +22,7 @@ import org.rust.stdext.zipValues
 
 open class Substitution(
     val typeSubst: Map<TyTypeParameter, Ty> = emptyMap(),
-    val regionSubst: Map<ReEarlyBound, Region> = emptyMap(),
```

Comment:
I suppose it was public on purpose in the same way as `typeSubst` and `constSubst`